llvm/test/MC/RISCV/align-non-executable.s

```diff
@@ -0,0 +1,29 @@
+# RUN: llvm-mc --filetype=obj --triple=riscv64 --mattr=+relax %s \
+# RUN:     | llvm-readobj -r - | FileCheck --check-prefixes=CHECK,RELAX %s
+# RUN: llvm-mc --filetype=obj --triple=riscv64 --mattr=-relax %s \
+# RUN:     | llvm-readobj -r - | FileCheck %s
+
+.section ".dummy", "a"
+.L1:
+  call func
+.p2align 3
+.L2:
+.dword .L2 - .L1
```

Comment:
Keep just `.dword .L2 - .L1` and remove other data directives to make the example minimum.